### DISCUSSION TITLE
Annotation Variations

### DISCUSSION THREAD

**Discussion by Meenakshise** [chartjs/chartjs-plugin-annotation#897] [Q&A]
Can I show the plotted point value on top of each plot as per the below screenshot?

![image](https://github.com/chartjs/chartjs-plugin-annotation/assets/5516586/0cae74c3-d9dc-4293-af30-a570f9d6681d)

Can I show the text in the chart instead of plotting a chart? 
![image](https://github.com/chartjs/chartjs-plugin-annotation/assets/5516586/8e1a7a7c-d138-42b8-91ff-9e66dbf5929a)

**Comment by stockiNail** [comment]
@Meenakshise in both cases the answer is yes, even if it's difficult to me to understand the use case and therefore maybe the annotation plugin is no the right solution.

For instance, the second use case could be addressed better by the [chartjs-chart-matrix](https://github.com/kurkle/chartjs-chart-matrix).

Staying on the annotation plugin, I think:

- the orange and yellow area could be box annotations
- the lines could be line annotations

I don't know if you have already a sample. If yes, can you share it? Otherwise let me take time to prepare a sample.

**Reply by stockiNail** [reply]
See answer to Chart.js issue: https://github.com/chartjs/Chart.js/issues/11332#issuecomment-1580175807
